userguide/directoryservices.rst

```diff
@@ -115,7 +115,7 @@ advanced options.
    |                          | menu          |          |                                                                                                                               |
    |                          |               |          |                                                                                                                               |
    +--------------------------+---------------+----------+-------------------------------------------------------------------------------------------------------------------------------+
-   | Certificate              | drop-down menu| ✓        | Select the certificate of the Active Directory server if  SSL connections are used. If a certificate does not exist, create   |
+   | Certificate              | drop-down menu| ✓        | Select the certificate of the Active Directory server if SSL connections are used. If a certificate does not exist, create    |
    |                          |               |          | a :ref:`Certificate Authority <CAs>`, then create a certificate on the Active Directory server. Import the certificate to the |
    |                          |               |          | %brand% system with the :ref:`Certificates` menu.                                                                             |
```

Comment:
```suggestion
    |                          |               |          | %brand% system using the :ref:`Certificates` menu.                                                                             |
```

Reply:
Fixing in local branch to avoid malformed table errors. Thanks for the suggestion!